src/main/java/com/github/platymemo/alaskanativecraft/AlaskaNativeCraftClient.java

```diff
@@ -34,6 +40,7 @@ public void onInitializeClient() {
         registerItemColors();
         registerBlockRenderLayers();
         registerHarpoonPacket();
+        AlaskaNativeCraftModels.registerEntityModels();
```

Comment:
I think `registerEntityModels()` should be moved here, to maintain parity with all the other methods. Otherwise, I'd want to break those ones into their respective classes (BE renderers in `AlaskaBlocks`, harpoon packet in `HarpoonEntity`, etc.)

Reply:
Moved `registerEntityModels()` in 96866f8db0c9766c89d1fa1fa38955d3992bfc4e